submission/distributor_test.go

```diff
@@ -35,15 +35,15 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 )
 
-func buildStubLogClient(log *loglist.Log) (client.AddLogClient, error) {
-	return buildRootedStubLC(log, RootsCerts)
+func localStubLogClient(log *loglist.Log) (client.AddLogClient, error) {
```

Comment:
Maybe this should also start 'new', as it has the same function signature as the others below?

func newLocalStubLogClient(...) (client.AddLogClient, error) { ... }

Reply:
Done.